hack/verify-generated-files-remake.sh

```diff
@@ -286,12 +286,32 @@ if [[ -n "${X}" ]]; then
 fi
 
 #
-# Test when we touch a file in a package that needs codegen.
+# Test when we touch a file in a package that needs codegen for all openapi specs.
+#
+
+assert_clean
+
+touch "staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go"
+touch "${STAMP}"
+make generated_files >/dev/null
+X="$(newer openapi "${STAMP}")"
+if [[ -z "${X}" || ${X} != "./pkg/generated/openapi/zz_generated.openapi.go
+./staging/src/k8s.io/apiextensions-apiserver/pkg/generated/openapi/zz_generated.openapi.go
+./staging/src/k8s.io/code-generator/_examples/apiserver/openapi/zz_generated.openapi.go
+./staging/src/k8s.io/sample-apiserver/pkg/generated/openapi/zz_generated.openapi.go" ]]; then
+    echo "Wrong generated openapi files changed after touching src file:"
+    echo "  ${X:-(none)}" | tr '\n' ' '
+    echo ""
+    exit 1
+fi
+
+#
+# Test when we touch a file in a package that needs codegen for only the main openapi spec.
 #
 
 assert_clean
 
-touch "staging/src/k8s.io/api/core/v1/types.go"
+touch "staging/src/k8s.io/api/apps/v1/types.go"
```

Comment:
why the change from core to apps here?

Reply:
Because core/v1 is a dependency for autoscaling/v1, which is needed by apiextensions-apiserver (in a follow-up). apps/v1 is only needed by kube-apiserver, and therefore is a better choice here.